resources/application-connector/charts/connector-service/files/certs-backup.sh

```diff
@@ -0,0 +1,27 @@
+set -e
+set -o pipefail
+
+backup() {
+  namespace="$1"
+  secretName="$2"
+  backupSecretName="$secretName-backup"
+
+  echo "Checking existence of $backupSecretName"
```

Comment:
You can easily replace 3 next lines with just one line delete secret $backupSecretName" --ignore-not-found